maat-court-data-api/src/main/java/gov/uk/courtdata/model/laastatus/LaaStatusUpdate.java

```diff
@@ -2,14 +2,13 @@
 
 import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.NoArgsConstructor;
+import lombok.*;
 
 
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
+@Data
```

Comment:
The object can be build using Builder and data annotation will allow for modification after creation which we dont want when generate json requests. Could we remove the @Data annotation, please?

Reply:
Changed to @Value.